streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStore.java

```diff
@@ -39,27 +39,62 @@
     private StoreChangeLogger<K, V> changeLogger;
     private StoreChangeLogger.ValueGetter<K, V> getter;
 
+    private boolean isInitialized = false;
```

Comment:
@guozhangwang The memory ones do not do any initialisation in init. Also RocksDBWindowedStore uses RocksDBStore for the segments, so each of the latter is initialised lazily.